### DISCUSSION TITLE
JingOS ARM V1.1.1

### DISCUSSION THREAD

**Discussion by JingOS-team** [JingOS-team/JingOS#80] [Announcements]
# Download from Google Drive:
https://drive.google.com/file/d/1FpY3uIs57ucddePiAbaeHx94SUTGgwQf/view

# General Improvements

Supporting hardware decode for local videos

Supporting WPA2/WPA3

Supporting “single click and swipe” and “double finger click”

Supporting MTP in File Manager

Optimising the performance of visual and audio plugins

# JingPad A1 Android App Support V0.7 Release Note
N.B. The Packages of Android App Support (JAAS) has been integrated into the OS. However, you still need to use the following commands to install the environment. Do not install other android containers if you wish to test JAAS.

Supporting 4 Inputting Methods including keyboard, trackpad, touchscreen and JingPad Pencil

Supporting input method

Solving known issues

Black screen after relaunching apps

Icons fail to display in the Quick Control Panel

Displaying issue in portrait mode (It has been changed to forced horizontal display)

<hr /><em>This discussion was created from the release <a href='https://github.com/JingOS-team/JingOS/releases/tag/ARMV1.X'>JingOS ARM V1.1.1</a>.</em>

**Comment by Halabikk** [comment]
The download link does not work.
